src/zkscript/bilinear_pairings/model/pairing.py

```diff
@@ -115,6 +119,7 @@ def single_pairing(
     def triple_pairing(
         self,
         modulo_threshold: int,
+        verify_gradients: int = 7,
```

Comment:
I see your point in having the bitmask, but maybe it could be easier to have a list of bool? (for consistency with the other calls of verify_gradients

Reply:
Addressed.